docs/input.md

```diff
@@ -465,6 +537,15 @@ Note that you should use `min`/`max` to set the hardcoded limits for the range i
 If you specify the `start` as a single (numerical) value, Range will return a single **number**. If you specify the `start` as an array of two (numerical) values, Range will return an **array of numbers** with two values.
 Note that the `start` value and the data of this input's value **must** be of the same type. This input will error if `start` is an array and the value passed from the component data is a number or vice versa.
 
+### Range Events that can be attached to Kiln-Input using kiln.js and the order in which they occur.  Each returns the value of the range as they occur
+
+* **start**
+* **slide**
+* **update**
+* **change**
+* **set**
+* **end**
```

Comment:
Not sure if you've taken into account to describe these events

Reply:
👍